integration/library_test.go

```diff
@@ -270,7 +270,9 @@ func checkPackageFromCommands(t *testing.T, actualFiles extractor.FileMap, osFou
 func checkPackages(actualFiles extractor.FileMap, t *testing.T, tc testCase) {
 	actualPkgs, err := analyzer.GetPackages(actualFiles)
 	require.NoError(t, err)
-	data, _ := ioutil.ReadFile(fmt.Sprintf("testdata/goldens/%s.expectedpackages.golden", strings.ReplaceAll(tc.imageName, "/", "")))
+	goldenFile := fmt.Sprintf("testdata/goldens/%s.expectedpackages.golden", strings.ReplaceAll(strings.ReplaceAll(tc.imageName, "/", ""), ":", ""))
```

Comment:
nit: I prefer strings.Replacer https://golang.org/pkg/strings/#Replacer.Replace

Reply:
thanks! I actually wanted to do something like that, didn't know about it. I will use it.